mods/cnc/fluent/chrome.ftl

```diff
@@ -459,7 +459,7 @@ button-server-lobby-start-game = Start Game
 
 ## mainmenu-prompts.yaml
 label-mainmenu-introduction-prompt-title = Establishing Battlefield Control
-label-bg-desc-a = Welcome back Commander! Initialize combat parameters using the options below.
```

Comment:
😢

Reply:
"Welcome back, Commander" is an iconic line across the series. Having it here targets that nostalgia